tests/test_dsr.py

```diff
@@ -42,3 +42,33 @@ def test_post_dsr_api_invalid(dsr_data):
 
     response = client.post("/dsr", data=json.dumps(dsr_data))
     assert response.status_code == 422
+
+
+def test_get_dsr_api(dsr_data):
+    """Tests DSR data GET method."""
+    data_1 = dsr_data.copy()
+    data_2 = dsr_data.copy()
+    data_2["Name"] = "Foo"
+    client.post("/dsr", data=json.dumps(data_1))
+    client.post("/dsr", data=json.dumps(data_2))
+
+    response = client.get("/dsr")
+    assert response.json()["data"] == dt.dsr_data
+
+
+def test_dsr_api_get_query(dsr_data):
```

Comment:
We can probably adequately test this DSR GET method without using the actual `dsr_data` fixture in it. The GET method is just getting an entry from a list, so we can pu anything we like there, and check that the list entry returned is the correct one. This will save time in the tests and probably fix the current issue in CI.